### PR TITLE
Add routing for articles

### LINKED ISSUE #610: Add Persian article routes
**Is your feature request related to a problem? Please describe.**
We do not have any routes for the Persian articles.

**Describe the solution you'd like**
To update the routes routes to be `/:service/articles/:id` and `/:service/articles/amp/:id` and then to be able to handle `news` and `persian` services.

Note: routes are defined in this file `/src/app/routes.index.js`

### PR DESCRIPTION
**Pull request by thekp**
Resolves #610 

**PR Changes:**
Updated the routing logic to accept a "service" param that is validated in the article's container class against a list of services. 
Refactored the "getInitialProps" function to be suitable for code climate

**Testing instructions:** 
"news" & "persian" are the only valid service params.
Please check that the validator works by visiting these links like these (locally): 

http://localhost:7080/khoa/articles/notvalid
http://localhost:7080/news/articles/notvalid
http://localhost:7080/khoa/articles/c0000000027o
http://localhost:7080/khoa/articles/amp/c0000000027o
http://localhost:7080/persian/articles/amp/c0000000027o
http://localhost:7080/news/articles/amp/c0000000027o

http://localhost:7080/news/articles/c0000000027o
http://localhost:7080/news/articles/amp/c0000000027o
http://localhost:7080/persian/articles/c0000000028o
http://localhost:7080/persian/articles/amp/c0000000028o

- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by sareh — DISMISSED** [review]
👍 Great work! Looks good.

File: src/app/containers/Article/index.test.jsx [hunk 1]
```diff
@@ -135,10 +143,12 @@ describe('ArticleContainer', () => {
     });
 
     describe('Validate route parameter ', () => {
-      it('to check the id is invalid before returning an empty object', async () => {
+      it('checks the id is invalid before returning an empty object', async () => {
         jest.spyOn(global.console, 'log');
         const invalidIdParam = 'route-21';
-        const invalidContext = { match: { params: { id: invalidIdParam } } };
+        const invalidContext = {
+          match: { params: { id: invalidIdParam, service: 'news' } },
```
Comment (jtart) [inline src/app/containers/Article/index.test.jsx]: This should use the `defaultServiceParam`.

**Review by jtart — CHANGES_REQUESTED** [review]

File: src/app/containers/Article/index.jsx [hunk 2]
```diff
@@ -5,20 +5,36 @@ import MainContent from '../MainContent';
 import articlePropTypes from '../../models/propTypes/article';
 import isAmpPath from '../../helpers/isAmpPath';
 
+const validateService = service => {
+  const services = ['news', 'persian'];
```
Comment (jtart) [inline src/app/containers/Article/index.jsx:9]: NB: this won't be defined here in the long-term, but it is fine for now.

**Review by jtart — DISMISSED** [review]

**Review by jtart — APPROVED** [review]

**Review by sareh — APPROVED** [review]
👍 Looks good!

**Pull request merged**